.github/workflows/site-deploy.yml

```diff
@@ -0,0 +1,72 @@
+# When publish a release. This workflow will trigger and deploy to site.
+
+name: Deploy website
+on:
+  release:
```

Comment:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token